.github/workflows/ci.yml

```diff
@@ -430,6 +431,7 @@ jobs:
       with:
         toolchain: 1.70.0
     - name: reset registry
+      if: runner.os != 'Windows'
       run: rm -fr ~/.cargo/registry
```

Comment:
I think we can remove the `rm` line? It was added to solve the `syn` issue, which is now fixed in another way.

Reply:
Good point!